docs/babel.md

```diff
@@ -0,0 +1,36 @@
+## Usage with Babel
```

Comment:
@greggb @tkh44 there are quite a few babel options (e.g. sourcemaps, inline etc etc).. I think they should be documented here..

Reply:
Absolutely - I'm not sure of all the options